ocs_ci/ocs/platform_nodes.py

```diff
@@ -93,20 +124,49 @@ def get_node_by_attached_volume(self, volume):
             "implemented for VMWare"
         )
 
-    def stop_nodes(self, nodes, wait=True):
-        raise NotImplementedError(
-            "Stop nodes functionality is not implemented for VMWare"
+    def stop_nodes(self, nodes, force=True):
+        """
+        Stop vSphere VMs
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            force (bool): True for force VM stop, False otherwise
+
+        """
+        vms = self.get_vms(nodes)
+        assert vms, (
+            f"Failed to get VM objects for nodes {[n.name for n in nodes]}"
         )
+        self.vsphere.stop_vms(vms, force=force)
 
     def start_nodes(self, nodes, wait=True):
-        raise NotImplementedError(
-            "Start nodes functionality is not implemented for VMWare"
+        """
+        Start vSphere VMs
+
+        Args:
+            nodes (list): The OCS objects of the nodes
```

Comment:
'wait' is not documented but also not used

Reply:
thanks, I'll remember to add it